backend/README.md

```diff
@@ -8,12 +8,12 @@ The current environment is initialized through the environment variable `ASPNETC
 
 When running the solution locally it applies the configuration setting in the following order;
 
-1. appsettings.json
-2. appsettings.`[environment]`.json
-3. UserSecrets `(if environment=Development)`
-4. Environment Variables
+1. appsettings.json (within /api)
+2. appsettings.`[environment]`.json (within /api)
+3. UserSecrets `(if environment=Development)` (within /api and /dal)
+4. Environment Variables (within /api and /dal)
 
-To run the solution with docker-compose create a .env file and populate it with the following;
+To run the solution with docker-compose create a .env file within the /dal and /api directories and populate each with the following;
 
 ```conf
 ASPNETCORE_ENVIRONMENT=Development
```

Comment:
This shouldn't be needed within the `/dal` project.  The `/dal` project can use the `connectionstrings.json` or a `.env` to configure the connection string used by `dotnet ef database update`.